From: Bruce Rogers <brogers@suse.com>
Date: Tue, 28 Apr 2020 09:53:49 -0600
Subject: docs: add SUSE support statements to html docs

Include-If: %if %{legacy_qemu_kvm}

We can fairly easily produce an html version of our support statements.
Now that qemu includes fairly good html-based documentation, leverage it
to expose our SUSE specific in-package support documentation.

Signed-off-by: Bruce Rogers <brogers@suse.com>
---
 docs/index.html.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/docs/index.html.in b/docs/index.html.in
index 6736fa4360cfb8c40cbab2a362b0..ad431321ee858bcbe3e237e687b5 100644
--- a/docs/index.html.in
+++ b/docs/index.html.in
@@ -7,6 +7,7 @@
     <body>
         <h1>QEMU @@VERSION@@ Documentation</h1>
         <ul>
+            <li><a href="/usr/share/doc/packages/qemu-kvm/kvm-supported.html">SUSE Support Statements</a></li>
             <li><a href="system/index.html">System Emulation User's Guide</a></li>
             <li><a href="user/index.html">User Mode Emulation User's Guide</a></li>
             <li><a href="tools/index.html">Tools Guide</a></li>
